components/login/login_controller/login_controller.jsx

```diff
@@ -8,15 +8,15 @@ import {Link} from 'react-router-dom';
 
```

Comment:
A lot of formatting changes in this file in this PR not related to the changes expected from the PR. A lot of styling changes in the code.